org.eclipse.transformer.cli/src/main/java/org/eclipse/transformer/cli/TransformerLoggerFactory.java

```diff
@@ -165,6 +165,12 @@ private void setLoggingProperties(String loggerName) {
 			setLoggingProperty(LoggerProperty.LOG_LEVEL_PREFIX + loggerName, "error");
 		}
 
+		// this is a shortcut to avoid bellow properties
+		if (settings.isShowDateTime) {
```

Comment:
Since this is done before the processing of settings.properties, the other settings.properties can override this.

Reply:
Yes, hopefully this is consistent with other properties